### DISCUSSION TITLE
How to copy images from one collection to another?

### DISCUSSION THREAD

**Discussion by jramiroz98** [pocketbase/pocketbase#4812] [Q&A] [answered]
Hello there! previously I built a "collection" that has an image per row, but now I want to use multiple images, hence I built a new collection called "images collection". I would like to copy the images from the "collection" and create entries in the "image collection"
Im sort of migrating the structure of the databases. In the future the "collection" wont have images, but rather the images will be called from the "images collection" allowing for multiple images

**Comment by ganigeorgiev** [comment]
It is not clear what actually is the the desired behavior. If you just want to change from single to multiple images, you can adjust the field option of your existing collection.

There is no automated way to copy/move uploaded files as it will depend on your old and new collections structure.

If you are still not able to resolve your issue please elaborate more clearly what you are trying to do (with details about the old and new collections) and whether you are using Go or JS migrations.

**Reply by jramiroz98** [reply]
I have an existing collection, and would like to create a new collection with the images the first one has

**Reply by jramiroz98** [reply]
What im currently doing is getting the image from the url from the first collection, in order to create a new entry in the new collection with it. Using JS and Sveltekit

**Comment by jramiroz98** [comment, marked as answer]
Found a solution
Pocketbase form data takes in Blobs
Therefore i Implemented the following function:
```
async function getImageBlob(imageUrl) {
    try {
        // Fetch the image data
        const response = await fetch(imageUrl);
        const blob = await response.blob();
        return blob;
    } catch (error) {
        console.error('Error fetching image:', error);
        throw error;
    }
}
```

then In the form data i did:
`formData.append('image', await getImageBlob(imgUrl));`
